templates/main.html

```diff
@@ -49,6 +49,9 @@ <h3>
         </div>
       {{/orgs}}
     </div>
+    <h4>Organization with no listed leader</h4>
+    <hr>
+    <div class="orgs" id="no-leader"></div>
```

Comment:
We can generate this on build time,
so that it works without client side js.

Reply:
Also do check if there's no organization without leaders,
don't show the heading/line "organization with no listed leaders"